src/components/Onfido/index.js

```diff
@@ -0,0 +1,70 @@
+import './index.css';
+import React from 'react';
+import * as OnfidoSDK from 'onfido-sdk-ui';
```

Comment:
Just to clarify, we are using another dependency here `onfido-sdk-ui` for web, and `@onfido/react-native-sdk` for native, right? Presumably because the RN sdk is not compatible with react-native-web?

Reply:
Yep, that's correct and how I understood it. I am pretty sure that Plaid is the same way.

Reply:
>the RN sdk is not compatible with react-native-web

Yep correct there's no web version in the SDK